src/components/Calculator.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import Output from './Output';
+import ButtonsBox from './ButtonsBox';
+import calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const [calculation, setCalculation] = useState({
```

Comment:
- [ ] The Calculator.js component should be a functional component.
```// ./src/components/Calculator.js

import React from "react"

function Calculator() {
  return <div>{/* Calculator app design goes here */}</div>
}

export default Calculator // export Calculator component
```

Reply:
You can also use the arrow function syntax to create functional components: I do this this component is functional